Modules/socketmodule.c

```diff
@@ -1097,23 +1097,35 @@ setipaddr(const char *name, struct sockaddr *addr_ret, size_t addr_ret_size, int
 
 
 /* Create a string object representing an IP address.
```

Comment:
Remove this comment, it's outdated.

Add a comment to `make_ipv4_addr` "Convert IPv4 sockaddr to a Python str."  Same for `make_ipv6_addr`.